purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterImpressionEvent.kt

```diff
@@ -0,0 +1,39 @@
+package com.revenuecat.purchases.customercenter.events
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.common.events.FeatureEvent
+import dev.drewhamilton.poko.Poko
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Type representing a customer center event and associated data. Meant for RevenueCatUI use.
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+@Poko
+class CustomerCenterImpressionEvent(
+    val creationData: CreationData = CreationData(),
+    val data: Data,
+) : FeatureEvent {
+    @ExperimentalPreviewRevenueCatPurchasesAPI
```

Comment:
NABD but this shouldn't be needed in nested classes of already `@ExperimentalPreviewRevenueCatPurchasesAPI`.